src/main/java/seedu/address/logic/commands/AddStudentScoreCommand.java

```diff
@@ -20,8 +26,19 @@ public class AddStudentScoreCommand extends Command {
     public static final String COMMAND_WORD = "addStuScore";
     // to expand
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a student score to the database.\n"
```

Comment:
Thanks for updating the message for this! Just to note though, the user won't be able to use this command anymore although we do use it internally.